scripts/wishlist/api.js

```diff
@@ -0,0 +1,96 @@
+import { getSignInToken, performMonolithGraphQLQuery } from '../commerce.js';
+
+const redirectToSignin = () => {
+  window.location = '/customer/login';
+};
+
+const getWishlistsQuery = `
+ query GetWishlists {
+  customer {
+    wishlists {
+      id
+      name
+      items_count
+      items_v2 {
+        items {
+          id
+          product {
+            uid
+            name
+            sku
+          }
+        }
+      }
+    }
+  }
+}
+`;
+
+const addProductToWishlistMutation = `
+mutation(
+  $wishlistId: ID!,
+  $sku: String!
+) {
+  addProductsToWishlist(
+    wishlistId: $wishlistId,
+    wishlistItems: [
+      {
+        sku: $sku
+        quantity: 1
+      }
+    ]
+  ) {
+    user_errors {
+      code
+      message
+    }
+    wishlist {
+      name
+    }
+  }
+}
+`;
+
+export async function getWishlists() {
+  const token = getSignInToken(redirectToSignin);
```

Comment:
Similar to line 64, it would make sense to do a login check here and don't even send the query if the user is not logged in. For this, maybe we can change `getSignInToken` to return `false` by default, when not implemented.

Reply:
It also works with the current implementation, `""` is false.

Good point though, pushed a new commit.